pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -727,7 +727,8 @@ func validateLiveMigration(field *k8sfield.Path, spec *v1.VirtualMachineInstance
 		})
 	} else if evictionStrategy != nil {
 		if *evictionStrategy != v1.EvictionStrategyLiveMigrate &&
```

Comment:
Can you factor this out into a function so that it reads better?
Something like
```go
if isValidEvictionStrategy(evictionStrategy) {
    causes = append(causes, metav1.StatusCause{
        Type:    metav1.CauseTypeFieldValueInvalid,
	Message: fmt.Sprintf("%s is set with an unrecognized option: %s", field.Child("evictionStrategy").String(), *spec.EvictionStrategy),
	Field:   field.Child("evictionStrategy").String(),
    })
}